devhelp/Makefile.am

```diff
@@ -0,0 +1,10 @@
+include $(top_srcdir)/build/vars.auxfiles.mk
+
+if GP_GTK3
```

Comment:
What's the use of this switch? :D

Reply:
The plugin used an old devhelp library. 
That's why this condition exists

Reply:
Well, I meant that both branches of the switch seem to do the same and so the switch is not necessary at all.

Additionally, Geany-Plugins are always using GTK3, we do not support GTK2 anymore.